src/typescript/azure-openapi-validator/rules/PathParameterMustNotBeEmpty.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const PathParametersMustNotBeEmpty: string = "PathParametersMustNotBeEmpty";
+
+rules.push({
+  id: "R2067",
+  name: PathParametersMustNotBeEmpty,
+  severity: "error",
+  category: "RPCViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.arm,
+
+  appliesTo_JsonQuery: "$.paths..parameters[?(@.in=='path')]",
+  run: function* (doc, node, path) {
+
+    const pathNodes: string[] = (<string>path[1]).split('/');
```

Comment:
nit: I don't remember how this works but I remember putting [this](https://github.com/vladbarosan/azure-openapi-validator/blob/0b78dff4cbd2cb851bcf9e445d8f30939f2ab987/src/typescript/azure-openapi-validator/rules/PostOperationIdContainsUrlVerb.ts#L17) comment , please make sure `path[1]` is the right thing to do

Reply:
Yes it is correct. the actual path is always the second element. In that case you referenced it works because it only has 2 elements so the last element is actually path[1].

Reply:
fun fact: variable not used (and I wouldn't know why) 😉

Reply:
yup don't needed it anymore